apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/boot/ServiceManager.java

```diff
@@ -36,11 +38,13 @@
     INSTANCE;
 
     private static final ILog LOGGER = LogManager.getLogger(ServiceManager.class);
+    public static Instrumentation INSTRUMENTATION;
     private Map<Class, BootService> bootedServices = Collections.emptyMap();
 
     public void boot() {
         bootedServices = loadAllServices();
 
+        JVMUtil.loadJvmInfo();
```

Comment:
Why called here?

Reply:
It seems that this position is not suitable. Do you have any suggestions?

Reply:
This is a static method, it could be anywhere you like, what do you mean?